dev-docs/Cloud-Env-Testing.md

```diff
@@ -45,6 +45,10 @@ Follow these steps to run the workflow:
 
     - **`run-sanity-tests`** (**optional**): Set to `true` to run sanity tests after the environment is set up. Default: `false`
 
+    - **`run-ui-sanity-tests`** (**optional**): Set to `true` to run Kibana UI sanity tests after the environment is set up. Default: `false`
```

Comment:
Maybe link the tests docs here (.github/actions/kibana-ftr/README.md)

Reply:
Linked to the `kibana-ftr` action documentation